nautobot/project-static/js/tableconfig.js

```diff
@@ -39,7 +39,12 @@ $(document).ready(function() {
             data: JSON.stringify(data),
         }).done(function () {
             // Reload the page
-            window.location.reload(true);
+            if (window.location.href.includes("saved_view=")){
+                var new_url = window.location.href+"&table_changes_pending=true";
```

Comment:
This is fragile, right? If the user removes "table_changes_pending=true" from their URL, then it reverts back to the saved-view despite the fact that their user-config has been updated in the DB? I don't immediately have an idea for a better approach but I think it's worth some thought.

Reply:
Yeah, the problem is how to represent that intermediate state (e.g. after the user made changes and before the user updated or saved those changes to a saved view) and render the intermediate state table. I agree this is fragile but I think this might be our best bet given how BaseTable is rendered.
Since user_config will be deprecated eventually, it could just be used as an intermediate saved point for table config and such.